applications/DeepSpeed-Chat/data/.gitignore

```diff
@@ -0,0 +1 @@
+*.json
```

Comment:
We even don't have this data/ directory in repo, so please not include this file in this PR.